### ISSUE TITLE
chore: remove unimplemented OptimizedConfiguration and SuboptimalConfiguration CSV validator entries

### ISSUE THREAD

**Issue by comnam90** [VeeamHub/veeam-healthcheck#100]
## Problem

`CCsvValidator` registers two CSV keys that have no backing implementation anywhere in the codebase:

```csharp
// CCsvValidator.cs
{ "OptimizedConfiguration", CsvValidationSeverity.Warning },
{ "SuboptimalConfiguration", CsvValidationSeverity.Warning },
```

There is no corresponding:
- `Export-VhcCsv` call in any collection `.ps1` script
- C# CSV info class, parser method, data former, or HTML table renderer

This causes spurious warnings on every VBR report run:

```
WARN [CsvValidator] WARNING: CSV file 'OptimizedConfiguration' not found at expected location.
WARN [CsvValidator] WARNING: CSV file 'SuboptimalConfiguration' not found at expected location.
```

## Proposed fix

Remove the two entries from `CCsvValidator.ExpectedVbrFiles` to eliminate the noise.
